.github/workflows/test.yml

```diff
@@ -161,10 +169,19 @@ jobs:
             django: 'Django>=4.2,<4.3'
             experimental: false
             parallel: '--parallel'
+          - python: '3.8'
+            django: 'Django>=3.2,<3.3'
+            experimental: false
+            mysql: 'mysql:8.1'
+          - python: '3.10'
+            django: 'Django>=4.2,<4.3'
+            experimental: false
+            parallel: '--parallel'
+            mysql: 'mysql:8.1'
```

Comment:
Same here, MySQL tests are always slower than the others, we could probably just make it so that the two existing configs in the matrix use 8.0 and 8.1 each.